api/v1/storagecluster_types.go

```diff
@@ -137,11 +148,42 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageMode specifies a mode of the external storage
+type ExternalStorageMode string
+
+const (
+	// ModeOcsToOcs specifies a "ocs-to-ocs" mode of the external storage
+	ModeOcsToOcs ExternalStorageMode = "ocs-to-ocs"
+
+	// ModeStandaloneCeph specifies a "standalone-ceph" mode of the external storage
+	ModeStandaloneCeph ExternalStorageMode = "standalone-ceph"
+)
+
 // ExternalStorageClusterSpec defines the spec of the external Storage Cluster
-// to be connected to the local cluster
+// to be connected to the external cluster
 type ExternalStorageClusterSpec struct {
-	// +optional
+	//+kubebuilder:default:=true
+	// Enable enables the external mode
 	Enable bool `json:"enable,omitempty"`
+
+	//+kubebuilder:default:=standalone-ceph
+	// Mode describes the mode of the external storage
+	Mode ExternalStorageMode `json:"mode,omitempty"`
+
+	// ConnectionString describes the endpoint and UID of the provider
+	// server to connect to the API server
+	ConnectionString string `json:"connectionString,omitempty"`
+
+	//+kubebuilder:default:="1TB"
+	// RequestedCapacity defines the requested storage on the provider
+	RequestedCapacity string `json:"requestedCapacity,omitempty"`
```

Comment:
I think it will benefit us to have all capacity-related values to be defined using the k8s type for sizes and not just plain strings.

Reply:
done